blockdag/dag.go

```diff
@@ -1656,28 +1656,28 @@ func (dag *BlockDAG) getBlueBlocksBetween(startHash, stopHash *daghash.Hash, max
 	return reversedNodes, nil
 }
 
-// GetBlueBlocksHashesBetween returns the hashes of the blue blocks after the
-// provided start hash until the provided stop hash is reached, or up to the
+// BlueBlocksHashesBetween returns the hashes of the blue blocks after the
+// provided low hash until the provided high hash is reached, or up to the
 // provided max number of block hashes.
 //
 // This function is safe for concurrent access.
-func (dag *BlockDAG) GetBlueBlocksHashesBetween(startHash, stopHash *daghash.Hash, maxHashes uint64) ([]*daghash.Hash, error) {
+func (dag *BlockDAG) BlueBlocksHashesBetween(lowHash, highHash *daghash.Hash, maxHashes uint64) ([]*daghash.Hash, error) {
```

Comment:
I think the gramatically correct name would be `BlueBlockHashesBetween` (without the s in blocks).
The same for `blueBlocksHashesBetween`